cli/src/semgrep/rule_match.py

```diff
@@ -528,7 +528,10 @@ def to_app_finding_format(self, commit_date: str) -> out.Finding:
             # string instead of a ValidationState. Fix the monkey
             # patchable version if you want monkey patching to work.
             validation_state=self.match.extra.validation_state,
+            historical_info=self.match.extra.historical_info,
         )
+        print(f"hist={self.match.extra.historical_info}")
+        print(f"ret={ret}")
```

Comment:
```suggestion
```

Un-needed prints?

Reply:
Ah whoops. Thanks.

Reply:
Somehow these got removed and re-added? Git loving me today.